src/components/shared/Property.tsx

```diff
@@ -34,29 +35,33 @@ export const Property: React.FunctionComponent<IProperty> = ({ onGoToRef }) => {
   return (
     <>
       {schemaNode.subpath.length > 0 && shouldShowPropertyName(schemaNode) && (
```

Comment:
Could you assert `subpath.length > 0` here instead? I know `subpath` and `schemaNode.subpath` are the same, but this is something you need to check for, when looking at this at first I am worried if this is a potential OOB condition

Reply:
You didn't write this, I see. Well in that case it's fully optional. 😄